deploy/cr/xsite/static-internal/xsite_a.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: example-clustera
+spec:
+  replicas: 1
+  expose:
+    type: LoadBalancer
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: SiteA
+        expose:
+          type: None
+      locations:
+      - name: SiteA
+        host: example-clustera-site
```

Comment:
where host `example-clustera-site` comes from? It looks like `metadata.name` + `-site`. Will it resolve to some IP?

Reply:
This is the k8s Service name resolved to internal IP.
If we will setup cross-site between separate namespaces, the service will include namespace, i.e. `example-clustera-site.namespace-a.svc.cluster.local`